src/sentry/plugins/bases/notify.py

```diff
@@ -66,7 +66,7 @@ def get_plugin_type(self):
 
     def notify(self, notification):
         event = notification.event
-        return self.notify_users(event.group, event)
+        return self.notify_users(event.group, event, [r.label for r in notification.rules])
```

Comment:
Could we change this to kwargs?

```suggestion
        return self.notify_users(event.group, event, triggering_rules=[r.label for r in notification.rules])
```